README.md

```diff
@@ -6,23 +6,37 @@ likely to be accepted.
 
 ## This Site
 
-### Compiling
+### Installing
 
 This site is mostly compiled by magic (specifically
 [Jekyll](http://jekyllrb.com/) and [Sass](http://sass-lang.com/) which will
-have to be installed before you can do anything. The following commands will
-install the necessary software:
+have to be installed before you can do anything.
+
+Unless you have a good reason not to, you will probably want to use Bundler
+to install the ruby dependencies. Just run this from the root directory of
+this repo:
+
+    bundle install
```

Comment:
As you've added it to .gitignore, are you expecting this to be `bundle install --path vendor/bundle` or similar?